### PR TITLE
device/plugin: rename used to device-used in api response

### PR DESCRIPTION
**Pull request by atinmu**
Fixes: #1286
Signed-off-by: Atin Mukherjee <amukherj@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
add to whitelist

**Comment by atinmu** [comment]
retest this please

**Comment by Madhu-1** [comment]
@nigelbabu CI is failing
```
2:47:49 [gluster_glusterd2] $ /bin/sh -xe /tmp/jenkins9069851796459309957.sh
12:47:50 + SSID_FILE=/home/gluster/workspace/gluster_glusterd2/cico-ssid
12:47:50 ++ cat /home/gluster/workspace/gluster_glusterd2/cico-ssid
12:47:50 cat: /home/gluster/workspace/gluster_glusterd2/cico-ssid: No such file or directory
```

**Comment by Madhu-1** [comment]
retest this please

**Comment by atinmu** [comment]
retest this please

**Comment by atinmu** [comment]
@aravindavk please review?

File: plugins/device/api/resp.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ type Info struct {
 	VgName        string `json:"vg-name"`
 	AvailableSize uint64 `json:"available-size"`
 	ExtentSize    uint64 `json:"extent-size"`
-	Used          bool   `json:"used"`
+	Used          bool   `json:"device_used"`
```
Comment (aravindavk) [inline plugins/device/api/resp.go]: `s/device_used/device-used/`

Comment (atinmu) [reply plugins/device/api/resp.go]: Argh! How did I make such stupid mistake?

**Comment by Madhu-1** [comment]
@atinmu need to update commit message, still it says ` rename used to device_used in api response `

**Pull request merged**